spec/features/shelters/cannot_delete_with_approved_pets_spec.rb

```diff
@@ -9,6 +9,9 @@
     @shelter_1 = Shelter.create!(name: "Abby's Shelter", address: "123 Maine Street", city: "Denver", state: "CO", zip: "80210")
     @shelter_2 = Shelter.create!(name: "Alex's Shelter", address: "123 Maine Street", city: "Bangor", state: "ME", zip: "04401")
 
+    @review_1 = @shelter_2.reviews.create(title: "Bad shelter", rating: 1, content: "It had fleas")
+    @review_2 = @shelter_1.reviews.create(title: "Great shelter", rating: 1, content: "It had fleas")
```

Comment:
Added a second review belonging to a shelter that I was not deleting, so I could test the total number of reviews to make sure that the review was deleted when the shelter